### ISSUE TITLE
Handling for notification bounces

### ISSUE THREAD

**Issue by c-schmitz** [Kunena/Kunena-Forum#9757]
We have a very active and old community forum based on Kunena.
In the forum users can subscribe to a category. Sometimes users don't log in anymore and their email address gets outdated/invalid.
In that case Kunena keeps sending emails to that outdated email address, because there is no feedback loop.

These emails are getting bounced and are bad for our email server reputation.

**Describe the solution you'd like**
I would like to have an option to configure a bounce email address in Kunena and a feature to handle bounces with IMAP.

**Describe alternatives you've considered**
Roll my own plugin?

**Additional context**
An app sending emails out at that scale should have also a proper way to handle bounces.

**Comment by xillibit** [comment]
So, your idea is that the notification bounce should be send to an unique specific mail address (to have all in one mailbox) just in case if the user has an invalid mail address ?

**Comment by c-schmitz** [comment]
Yes, that's how most systems do it that send alof of emails and expect bounces.  But I see that would probably would be huge effort.

Alternative and simpler idea: 
Category subscriptions expire after a certain time if the user did not log in for x months/days. 
Advanced version:
x days before the expiration send out a reminder email asking to log in to keep the subscription

**Comment by Ruud68** [comment]
so just finished an extension for a customer where I read in a mailbox and handle incoming mails. This is not something that comes out-of-the php box but you need a package for that (php can only send emails, not receive). There are a lot of challenges especially when doing this as part of an extension where you do not have control over the inbox / mail server. oauth being one of them.

I thought there was something in place (but i can be mistaken) that there is only one email sent, a next email will only be sent to a user when this user actually logged in? Can be mistaken though.

Not having this 'issue' myself as I regularly do a mailing to customers, when a message bounces (a hard bounce, not a soft 'mailbox full' bounce), I just remove that account: for me that is part of maintenance.
